licenses/LICENSE-luaffi.txt

```diff
@@ -0,0 +1,58 @@
+BSD License
+
+For luaffifb software
```

Comment:
A thing, why do we have this update? I think we don't copy the luaffi source codes.

Reply:
ok